src/Employee.java

```diff
@@ -0,0 +1,20 @@
+public class Employee {
+    private String firstName;
+    private String secondName;
+    private String surname;
+    private String position;
+    private String email;
+    private Long phoneNumber;
```

Comment:
Why Long?

Reply:
It seemed to me that this was the right decision, since it is easier to store numbers in a database in a numeric format, and the integer is too small for a phone number

Reply:
How then the number "0088" will look like? Consider using String